src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1723,34 +1984,107 @@ namespace WaterToAirHeatPumpSimple {
+
+                                simpleWatertoAirHP.RatedCapCoolTotal = RatedCapCoolTotalDes;
+                                OutputReportPredefined::PreDefTableEntry(state,
+                                                                         state.dataOutRptPredefined->pdchWAHPDD,
+                                                                         state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).Name,
+                                                                         "Cooling");
+                            }
+                            // Set the global DX cooling coil capacity variable for use by other objects
+                            state.dataSize->DXCoolCap = simpleWatertoAirHP.RatedCapCoolTotal;
+                        } else if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionHeatingCoilNum).WAHPPlantType ==
+                                       DataPlant::PlantEquipmentType::CoilWAHPHeatingEquationFit &&
+                                   state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionHeatingCoilNum).RatedCapHeat ==
+                                       DataSizing::AutoSize) {
+                            // case 2: companion heating coil has not already been sized
+                            // we only pass the rated total cooling capacity determined
+                            // based on cooling design day which is used to decide if the
+                            // coil needs to be sized of the heating coil size
+                            //
+                            // no capcity adjustment based on system flow because the capacity could change
+                            // once the heating coil has been sized
+                            state.dataSize->DXCoolCap = RatedCapCoolTotalDes;
+                        } else if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionHeatingCoilNum).WAHPPlantType !=
+                                   DataPlant::PlantEquipmentType::CoilWAHPHeatingEquationFit) {
+                            // case 3: companion heating coil is not of the "equationfit" type and hence doesn't use the rated heating to cooling
+                            // coil capacity ratio
+                            // adjust for system air flow -- capacity is based on cooling design day calcs
+                            // adjust by ratio of system to cooling air flow rate and enthalpy delta across the coil at these different airflow
+                            RatedCapCoolTotalDes *= (RatedAirVolFlowRateDes / CoolingAirVolFlowRateDes) * dHratio;
+                            simpleWatertoAirHP.RatedCapCoolTotal = RatedCapCoolTotalDes;
+                            // Set the global DX cooling coil capacity variable for use by other objects
+                            state.dataSize->DXCoolCap = simpleWatertoAirHP.RatedCapCoolTotal;
+                        }
```

Comment:
Adjust cooling coil capacity based on heating companion coils. Comments in the code document the three different cases that are handled.